networkx/readwrite/tests/test_gml.py

```diff
@@ -571,10 +571,6 @@ def assert_generate_error(*args, **kwargs):
         G = nx.Graph()
         G.graph["data"] = frozenset([1, 2, 3])
         assert_generate_error(G, stringizer=literal_stringizer)
-        G = nx.Graph()
-        G.graph["data"] = []
-        assert_generate_error(G)
-        assert_generate_error(G, stringizer=len)
```

Comment:
Empty lists were not allowed in the original implementation, but I didn't see anything in the standard that said empty lists are not allowed:

https://web.archive.org/web/20190207140002/http://www.fim.uni-passau.de/index.php?id=17297&L=1

I may have missed something though.